apple/internal/partials/cc_info_dylibs.bzl

```diff
@@ -0,0 +1,70 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Partial implementation for gathering cc_info dylibs."""
+
+load(
+    "@bazel_skylib//lib:partial.bzl",
+    "partial",
+)
+load(
+    "@build_bazel_rules_apple//apple/internal:processor.bzl",
+    "processor",
+)
+
+def _cc_info_dylibs_partial_impl(
+        *,
+        embedded_targets):
+    """Implementation for the CcInfo dylibs processing partial."""
+    bundle_files = []
+
+    for target in embedded_targets:
+        if CcInfo in target:
```

Comment:
Can you change this to `if CcInfo not in target:` and have it `continue` to lower the nesting?

Reply:
Yep, done!